styles.css

```diff
@@ -0,0 +1,501 @@
+  width: 90%;
+  font-size: 15px;
+}
+
+.chev {
+  margin-left: 20px;
+}
+
+.artist_name,
+.artist_type,
+.artist_desc,
+.artist_rating {
+  font-size: 15px;
+}
+
+.artist_type {
+  color: #ec5242;
+}
+
+/* partner section */
+.partners {
+  background-color: #272a31;
+  color: #fff;
+}
+
+#partners-about {
+  display: flex;
+}
+
+#partners-index {
+  display: none;
+}
+
+.partner-1,
+.partner-2,
+.partner-3,
+.partner-4,
+.partner-5 {
+  /* width: 90px; */
+  margin: 20px;
+  align-self: center;
+}
+
+#partnerships-about {
+  display: flex;
+  width: 100%;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+/* about page  */
+.logo-about-page {
+  width: 150px;
+  height: 150px;
+}
+
+.line {
+  border-bottom: 1px solid lightgrey;
+  width: 90%;
+  justify-content: center;
+}
+
+#past_event_pics {
+  gap: 20px;
+  display: flex;
+  flex-direction: column;
+}
+
+/* footer  */
+.footer-logo {
+  width: 120px;
+  height: 120px;
+}
+
+.desktop-navbar {
+  display: none;
+}
+
+.fb,
+.twitter {
+  display: none;
+}
+
+.desk-menu {
+  display: none;
+}
+
+.black_nav_desktop {
+  display: none;
+}
+
+#footer-index {
+  display: none;
+}
+
+@media screen and (min-width: 768px) {
+  .hamburger,
+  .mobile-navbar {
+    display: none;
+  }
+
+  /* header and nav  */
+  header {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .fb,
+  .twitter {
+    display: flex;
+    -webkit-filter: grayscale(1) invert(1);
+    filter: grayscale(1) invert(1);
+  }
+
+  .nav_logos_desktop {
+    display: flex;
+    gap: 30px;
+  }
+
+  .black_nav_desktop {
+    display: flex;
+    width: 100%;
+    justify-content: flex-end;
+    background-color: #272a31;
+    padding: 10px;
+    gap: 50px;
+  }
+
+  .black_nav_desktop a {
+    text-decoration: none;
+    color: #fff;
+  }
+
+  .desk-menu {
+    display: flex;
+  }
+
+  .desk-menu a {
+    text-decoration: none;
+    color: #272a31;
+  }
+
+  #about-desk,
+  #cc {
+    color: #ec5242;
+  }
+
+  #cc {
+    border: 5px solid;
+    padding: 5px;
+  }
+
+  .menu {
+    display: flex;
+    order: 2;
+    width: 100%;
+    justify-content: space-around;
+    list-style: none;
+    background-color: #fff;
+    height: 4rem;
+    align-items: center;
+  }
+
+  .logo-desk {
+    width: 50px;
+    height: 50px;
+  }
+
+  /* welcome section  */
+  .welcome-container {
+    text-align: justify;
+    margin: 60px;
+    margin-top: 250px;
+    width: 60vw;
+    margin-left: 150px;
+  }
+
+  .welcome-top {
+    font-size: 35px;
+    color: #ec5242;
+  }
+
+  .welcome,
+  .break {
+    font-size: 45px;
+    color: rgba(255, 101, 11, 1);
+  }
+
+  /* event section  */
+  .event {
+    justify-content: center;
+    min-height: 500px;
+  }
+
+  .test > * {
+    width: auto;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+  }
+
+  .event_main {
+    width: 17.7%;
+    justify-content: space-evenly;
+    display: flex;
+    flex-direction: column;
+    background-color: rgba(255, 255, 255, 0.1);
+  }
```

Comment:
- [ ] Again as mentioned by the previous reviewer https://github.com/mnaibei/capstone_1/pull/1#discussion_r1133988011 please implement the hover effect for all the cards as shown in the design for example on hover border 1px white.

Reply:
Done